.vscode/settings.json

```diff
@@ -0,0 +1,83 @@
+{
+    "files.associations": {
+        "array": "cpp",
+        "hash_map": "cpp",
+        "hash_set": "cpp",
```

Comment:
This file should not have been added. Can you please fix the commit that introduced this so it never appears in the commit history?